repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/common/AuthenticationHandler.java

```diff
@@ -96,10 +97,16 @@ protected AuthenticationHandler(Configurables configurables, AuthTokenCache cach
 
     protected abstract AuthToken validateToken(ExtractorResult<String> account, String token) throws AuthServiceException;
 
+    protected abstract AuthToken validateToken(ExtractorResult<String> account, String token, String requestGuid) throws AuthServiceException;
```

Comment:
Why duplicate all the methods? I'm assuming it was with an eye towards backwards compatibility, but i don't like have the vestigial methods. Unless you've got a good reason why to keep them?